a3p-integration/proposals/a:upgrade-next/use.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# UNTIl this is upstream https://github.com/Agoric/agoric-3-proposals/issues/40
+# Set to zero so tests don't have to pay gas (we're not testing that)
+sed --in-place=.bak s/'minimum-gas-prices = ""'/'minimum-gas-prices = "0ubld,0uist"'/ ~/.agoric/config/app.toml
```

Comment:
```suggestion
# UNTIL this is upstream https://github.com/Agoric/agoric-3-proposals/issues/40
# Set to zero so tests don't have to pay gas (we're not testing that)
sed -i .bak -e 's/minimum-gas-prices = ""/minimum-gas-prices = "0ubld,0uist"/' ~/.agoric/config/app.toml
```
`-e` says the next arg is an expression. Single quotes can capture the entire substitution expression, which can then include spaces and doublequotes. `--in-place=.bak` didn't work on my version of sed. does `-i` work for you?

Reply:
> -e says the next arg is an expression. Single quotes can capture the entire substitution expression, which can then include spaces and doublequotes.

Sound like you think that would read better. I'll make those changes.

> --in-place=.bak didn't work on my version of sed. does -i work for you?

Your version like on your Mac? IIRC it didn't work in Ubuntu, where this line runs.

Reply:
> Your version like on your Mac?
Yeah. nothing special, just what's here.